lib/assessment.md

```diff
@@ -22,14 +22,23 @@ Using this format will make it easier for us to grade you, so that we can get ev
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
 <!-- your answer starts here -->
-
+The web is a service in that the client requests the webpage (or other online) content, and the server delivers it
 <!-- your answer ends here -->
 
 ## Question 2
 
 Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body.
 <!-- your answer starts here -->
-
+```HTML
+<html>
+  <header><h1>How to Succeed in Business Without Even Trying</h1></header>
+  <ul>
+    <li>Frequently state that you are soooooo busy, but you are loving the work</li>
+    <li>Dress better than your colleagues</li>
+    <li>Find a capable, early-career minion to handle your responsibilities (outsource if necessary)</li>
+  </ul>
+</html>
+```
```

Comment:
Not quite. Your document should be divided into `<head>` (for metadata like the page's title), and `<body>` (for the visible content of your page). Additionally, though it's not _strictly necessary_, you should generally include the `<!DOCTYPE>` tag at the top.